<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>411</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="downPushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>0</number>
    </property>
    <property name="autoRepeatInterval">
     <number>200</number>
    </property>
   </widget>
   <widget class="QPushButton" name="leftPushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightPushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameoverLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>GAME OVER!</string>
    </property>
   </widget>
   <widget class="QLabel" name="blockstextLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>111</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blocks on screen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="blocksnumberLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="hiscoreLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP PLAYERS:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="hiscoreTextBrowser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>256</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="yournameLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playernameLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submitscorePushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit score!</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeplayedLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time played: </string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0 min 0 sec</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string notr="true">Difficulty</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 (Easy)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 (Normal)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 (Hard)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 (Insane)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="dropPushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DROP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipPushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FLIP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
